docs/references/expo/offline-support.mdx

```diff
@@ -71,3 +71,49 @@ To enable offline support in your Expo app, follow these steps:
   }
   ```
 </Steps>
+
+## How to handle network errors
+
+To handle network errors in your Expo app, you can use the `isClerkRuntimeError()` function to check if the error is a Clerk-related error. Clerk-related errors are returned as an array of [`ClerkAPIError`](/docs/references/javascript/types/clerk-api-error) objects. These errors contain a `code` property that you can use to identify the specific error. See the following example.
```

Comment:
I'm curious, what's the difference between isClerkRuntimeError and isClerkAPIResponseError
in our [error handling guide](https://clerk.com/docs/custom-flows/error-handling), we use `isClerkAPIResponseError`

Reply:
@anagstef can confirm, but from a quick look at the code plus the name I think these are errors related to Clerk actually running in the app, versus Clerk interacting with the API. The network error would be runtime as its the app running but not being able to make a request to FAPI. `ClerkAPIError` would be an error returned from FAPI.

Reply:
ahh makes sense! thank you for the explanation ❤️

Reply:
Thanks for confirming!